src/ray/core_worker/actor_manager.h

```diff
@@ -125,6 +125,10 @@ class ActorManager {
   /// \param actor_id The actor id of the handle that will be invalidated.
   void OnActorKilled(const ActorID &actor_id);
 
+  /// Subscribe to the state of actor.
```

Comment:
We should mention this function is idempotent. Also, add one line between \param and the first line!